### PR TITLE
Olwa pr

### PR DESCRIPTION
**Pull request by oliver-walker-hms**

### CODE REVIEW

File: src/com/hms/flexyosisoftconnector/OSIsoftConfig.java [hunk 1]
```diff
@@ -93,6 +93,9 @@
   /** default http timeout in seconds for if it is not set through configuration file. */
   private static final int HTTP_TIMEOUT_SECONDS_DEFAULT = 2;
 
+  /** Default logging level for the SC logger library is set to 4 for 'info'. */
+  private static final int LOGGING_LEVEL_DEFAULT = 4;
```
Comment (alexjhawk) [inline src/com/hms/flexyosisoftconnector/OSIsoftConfig.java:97]: Is there a reason for this change? I think the logging should only really output warnings at most by default. If additional information is needed, it would be set via config file.

Comment (oliver-walker-hms) [reply src/com/hms/flexyosisoftconnector/OSIsoftConfig.java:97]: Having a sane default simplifies the configuration process. You might be right that Warning is a better default than info. I can update this change.

File: src/com/hms/flexyosisoftconnector/OSIsoftConfig.java [hunk 2]
```diff
@@ -162,11 +165,14 @@ public static void initConfig(String configFile) throws JSONException, IOExcepti
...
 
+    boolean res;
     if (appConfig.has("LoggingLevel")) {
-      boolean res = Logger.SET_LOG_LEVEL(appConfig.getInt("LoggingLevel"));
-      if (!res) {
-        Logger.LOG_SERIOUS("Invalid logging level specified");
-      }
+      res = Logger.SET_LOG_LEVEL(appConfig.getInt("LoggingLevel"));
+    } else {
+      res = Logger.SET_LOG_LEVEL(LOGGING_LEVEL_DEFAULT);
```
Comment (TomKimsey) [inline src/com/hms/flexyosisoftconnector/OSIsoftConfig.java:172]: Should there be a message here saying that there is no logging level in the config file?

File: .classpath [hunk 3]
```diff
@@ -2,5 +2,12 @@
 <classpath>
 	<classpathentry exported="true" kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
 	<classpathentry kind="src" path="src"/>
+	<classpathentry kind="src" path="lib/sc-flexy-datapoint-lib/src"/>
+	<classpathentry kind="src" path="lib/sc-flexy-file-util-lib/src"/>
+	<classpathentry kind="src" path="lib/sc-flexy-json-lib/src"/>
+	<classpathentry kind="src" path="lib/sc-flexy-logger-lib/src"/>
+	<classpathentry kind="src" path="lib/sc-flexy-string-lib/src"/>
+	<classpathentry kind="src" path="lib/sc-flexy-tag-data-lib/src"/>
+	<classpathentry kind="src" path="lib/sc-flexy-tag-info-lib/src"/>
```
Comment (TomKimsey) [inline .classpath:11]: Classpath changes should be a separate commit.

**Review by TomKimsey — CHANGES_REQUESTED** [review]

File: src/com/hms/flexyosisoftconnector/OSIsoftConfig.java (hunk 2 above)
Comment (oliver-walker-hms) [reply src/com/hms/flexyosisoftconnector/OSIsoftConfig.java:172]: added

**Review by alexjhawk — APPROVED** [review]

**Review by TomKimsey — APPROVED** [review]
